raptorWeb/raptorbot/discordbot/util/raptorbot_settings.py

```diff
@@ -0,0 +1,12 @@
+from os import getenv
+from dotenv import load_dotenv
+
+# Load .env file and load Bot Token from file
+load_dotenv()
+TOKEN = getenv('DISCORD_TOKEN')
+
+# Settings for the Discord Bot
+DESCRIPTION = "I do work for the ShadowRaptorMC website!"
```

Comment:
These fields in particular ought to be editable as a Model. some type of BotConfiguration model.

Reply:
I've decided these would best be placed in the project-wide .env file. They are core to the bot, and would run without gameservers existing.